examples/subplots_axes_and_figures/align_labels_demo.py

```diff
@@ -0,0 +1,38 @@
+"""
+===============
+Aligning Labels
+===============
+
+Aligning xlabel and ylabel using
+:func:`matplotlib.figure.align_xlabels` and
```

Comment:
see above re: syntax.  the functions don't actually exist in the `figure` *module*.